src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+  
+.stars {
+  display: flex;
+}
+  
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin: 2px;

```

Comment:
your 1st and last start in the row have margin too + you have a vertical margin 🥲